libs/rblibtorrent/Makefile

```diff
@@ -0,0 +1,73 @@
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=rblibtorrent
+PKG_VERSION:=1.1.12
```

Comment:
1.2 is out

Reply:
deluge can work unstable with rblibtorrent version 1.2. on my PC desktop, deluge has a <net-libs / libtorrent-rasterbar-1.2 dependency (gentoo distribution). I will update the rblibtorrent package to version 1.2 when version 1.2 stabilizes on my linux distribution